.travis.yml

```diff
@@ -14,7 +14,7 @@ addons:
       - make
 
 cache:
-  yarn: true
+  yarn: false
```

Comment:
just delete? i assume false is the default, or not?

Reply:
We don't use `.travis.yml` but I thought I change it anyways....
I guess we could remove the travis file completely, but that has nothing todo with change from yarn to npm, so not sure what to do...